openvdb/doc/changes.txt

```diff
@@ -6,6 +6,12 @@
 @par
 <B>Version 6.1.1</B> - <I>In development</I>
 
+@par
+Bug fixes:
+- Fixed a bug in @vdblink::tools::computeScalarPotential computeScalarPotential()
```

Comment:
One minor fix: Doxygen requires parentheses after the symbol name when the symbol is a function, so change this to
```
@vdblink::tools::computeScalarPotential() computeScalarPotential@endlink
```

Reply:
Nice catch thanks